doc-content/enterprise-only/openshift/con-dashbuilder-standalone.adoc

```diff
@@ -0,0 +1,8 @@
+[id="con-dashbuilder-standalone_{context}"]
+= Dashbuilder Standalone on {OPENSHIFT}
+
+Dashbuilder Standalone is an independent implementation of Dashbuilder, the dashboard and reporting tool integrated with {CENTRAL}.
+
+You can install Dashbuilder Standalone separately from {PRODUCT}. When you install Dashbuilder Standalone, you can import or create your own dashboards.
```

Comment:
Minor rewrite for minimalism. Up to you as long as I have retained your intended info.

Install Dashbuilder Standalone separately from {PRODUCT} to import or create your own dashboards.

Reply:
I updated it to "You can install Dashbuilder Standalone separately from {PRODUCT} and import or create your own dashboards."